lds/arm/task-rlink.x

```diff
@@ -55,6 +55,10 @@ SECTIONS
     __sheap = .;
   }
 
+  .eth_bulk (NOLOAD) : ALIGN(4) {
+    *(.eth_bulk .eth_bulk.*);
+  }
+
```

Comment:
My suspicion is that Oxide folks didn't add this section to `task-rlink.x` because they did not want to pollute the common linker scripts with sections that are required by only a specific task. That is why they probably went through the trouble of adding section mapping in `xtask` so that they don't have to worry about the section or the location. Given that, it might be difficult to get this change upstream.

Reply:
I agree. We could have the flag added only for riscv apps.

Reply:
That should be fine.